restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/RCDevice.java

```diff
@@ -1730,38 +1684,30 @@ public void registerForPush(boolean clearFcmData){
       //remove it
       if (!registerForPushAvailable) {
          if (isServiceAttached && listener != null) {
-            listener.onInitialized(this, cachedConnectivityStatus, RCClient.ErrorCodes.ERROR_DEVICE_PUSH_PARAMETERS_MISSING.ordinal(), RCClient.errorText(RCClient.ErrorCodes.ERROR_DEVICE_PUSH_PARAMETERS_MISSING));
+            listener.onWarning(RCClient.ErrorCodes.ERROR_DEVICE_PUSH_PARAMETERS_MISSING.ordinal(), RCClient.errorText(RCClient.ErrorCodes.ERROR_DEVICE_PUSH_PARAMETERS_MISSING));
```

Comment:
Now that I see the implementation I'm thinking we might need to do some more brainstorming on this; I think we can further improve in some areas :D. In any case, let's not change it at this point, as I'd like to also see it in action to get the full picture. If we decide to update we can do it as part of #767 that is very closely related and comes in next sprint

Reply:
ok